index.js

```diff
@@ -0,0 +1,21 @@
+import express, { Router } from "express";
+import bodyParser from "body-parser";
+import userRoute from "./routes/User.js";
+
+const app = express();
+const PORT = 3000;
+const rootUserRoute = Router();
+
+app.use(
+  bodyParser.urlencoded({
+    extended: false,
+  })
+);
+app.use(bodyParser.json());
+
+app.use("/api", rootUserRoute);
```

Comment:
Put this below your definition of `rootUserRoute` since you are using the variable inside of this call.